m2cgen/interpreters/c/interpreter.py

```diff
@@ -21,7 +21,9 @@ class CInterpreter(ToCodeInterpreter,
     power_function_name = "pow"
     tanh_function_name = "tanh"
 
-    def __init__(self, indent=4, *args, **kwargs):
+    def __init__(self, indent=4, function_name="score", *args, **kwargs):
+        self.function_name = function_name
```

Comment:
It should be used later in 
```
with self._cg.function_definition(
        name=self.function_name,
```

Reply:
My bad, I completely missed that.

Thanks for spotting this @StrikerRUS!